index.html

```diff
@@ -7,7 +7,7 @@
     </head>
     <body>
         <header>
-            Company Logo
+            Company Logo and image
```

Comment:
Please wrap this in a div 
I also added a suggestion to change image to Image
You can apply my suggestion by clicking on "commit" suggestion
Clicking on commit suggestion will commit the code and actually create a new commit
This means you will need to pull the latest changes from this branch
git pull origin feature/search-2
After that, add the requested changes (Add a div wrapper) and then commit and push

Come back here to see the changes 

```suggestion
            Company Logo and Image
```